InterfaceMATLAB/tsgMakeFourier.m

```diff
@@ -0,0 +1,221 @@
+function [lGrid, points] = tsgMakeFourier(sGridName, iDim, iOut, sType, iDepth, mTransformAB, vAnisotropy, sConformalMap, vConfromalWeights, vLimitLevels)
```

Comment:
* this needs testing is `tsgCoreTests()`, see how other grids are tested and adjust the tests. Keep things simple.
* make sure to add a test regarding the complex numbers and the matrix.